postgres/init/20-portal.sql

```diff
@@ -186,8 +186,13 @@ CREATE TABLE portal.apps (
     name character varying(255),
     date_released timestamp without time zone,
     thumbnail_url character varying(255),
+    thumbnail_url1 character varying(255),
+    thumbnail_url2 character varying(255),
+    thumbnail_url3 character varying(255),
     app_url character varying(255),
     marketing_url character varying(255),
+    contact_email character varying(255),
```

Comment:
Wouldn't we rather bundle all contact-related fields into a new entity for reusability? Or is this unique for each app?

Reply:
@wira-mms hard to say right now. Likely unique since there might be a Sales Person behind each app